R/to_rmd.R

```diff
@@ -0,0 +1,267 @@
+content_to_rmd <- function(content, output_dir, ..., include_results) {
+  if (include_results || isTRUE(attr(content, "keep"))) {
+    suppressWarnings(hashname <- rlang::hash(content))
+    hashname_file <- paste0(hashname, ".rds")
+    path <- tempfile(fileext = ".rds")
+    suppressWarnings(saveRDS(content, file = path))
+    file.copy(path, file.path(output_dir, hashname_file))
+    sprintf("```{r echo = FALSE}\nreadRDS('%s')\n```", hashname_file)
+  }
+}
+
+#' Convert `ReporterCard`/`doc` content to `rmarkdown`
+#'
+#' This is an S3 generic that is used to generate content in `rmarkdown` format
+#' from various types of blocks in a `ReporterCard` or `doc` object.
+#'
+#' @details
```

Comment:
Maybe a section with title instead
```suggestion
#' # Customise `to_rmd`
```